tee-worker/ts-tests/integration-tests/common/call.ts

```diff
@@ -25,7 +25,7 @@ export async function sendRequest(
     api: ApiPromise
 ): Promise<WorkerRpcReturnValue> {
     const rawRes = await wsClient.sendRequest(request, { requestId: 1, timeout: 6000 });
-    const res = api.createType('WorkerRpcReturnValue', rawRes.result) as unknown as WorkerRpcReturnValue;
+    const res: any = api.createType('WorkerRpcReturnValue', rawRes.result) as WorkerRpcReturnValue;
```

Comment:
@Verin1005 why `: any` and not `: WorkerRpcReturnValue`? :thinking:

Reply:
fixed!